src/content/docs/browser/new-relic-browser/configuration/filter-ajax-request-events.mdx

```diff
@@ -0,0 +1,24 @@
+---
+title: 'Filter AjaxRequest events'
+metaDescription: "For New Relic browser monitoring: use the app settings page to filter which network calls are recorded as AjaxRequest events."
+---
+In agent version v1211 or higher, all network requests made by a page are recorded as AjaxRequest events. Included with this change is a new deny list configuration option available in the application settings page to filter which requests record events.
+Regardless of this filter, all network requests are captured as metrics and available in the AJAX Page.
+
+To update the deny list of URLs your application will filter from creating events, go to the app settings page:
+1. Go to [one.newrelic.com](http://one.newrelic.com), and click **Browser**. 
+2. Select an app. 
+3. On the left navigation, click **App settings**.
+4. Under **Ajax Request Deny List**, add the filters you would like to apply to your app.
+
+   <Callout variant="important">
+     Requests can be blocked in three ways:
+
+      * To block recording of all AjaxRequest events, add the * wildcard. Example: `*`
+      * To block recording of AjaxRequest events to a domain, enter just the domain name. Example: `example.com`
+      * To block recording of AjaxRequest events to a specific domain and path, enter the domain and path. Example: `example.com/path`
+
+     The protocol, port, search and hash of a URL are ignored by the deny list.
+   </Callout>
+5. Select **Save application settings** to update the agent configuration.
+6. [Redeploy the browser agent](/docs/browser/new-relic-browser/installation/upgrade-browser-agent) (either restarting the associated APM agent or updating the copy/paste browser installation).
```

Comment:
This is great info! I am wondering, what happens next? After they set up their deny list and redeploy the agent, is there anything they should do? Like check the results in the UI? Run a query to see if their deny list is working the way the want? Or is everything good to go at this point? I'm not saying there must be a next step. If this is all they need to know, hooray, no need for a novel! :-)